<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SQLiteEditor</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="treeDBWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>240</width>
      <height>570</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Базы данных</string>
     </property>
    </column>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>731</width>
      <height>571</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>Столбцы</string>
     </attribute>
     <widget class="QTableView" name="table1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>41</y>
        <width>725</width>
        <height>501</height>
       </rect>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="colDel">
      <property name="geometry">
       <rect>
        <x>76</x>
        <y>4</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QToolButton" name="colEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>4</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>?</string>
      </property>
     </widget>
     <widget class="QToolButton" name="colAdd">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>4</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>fds</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>Данные</string>
     </attribute>
     <widget class="QTableView" name="table2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>41</y>
        <width>725</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>970</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="qCreateDB"/>
    <addaction name="qOpenDB"/>
   </widget>
   <addaction name="menufile"/>
  </widget>
  <action name="qCreateDB">
   <property name="text">
    <string>Создать базу данных</string>
   </property>
  </action>
  <action name="qOpenDB">
   <property name="text">
    <string>Открыть базу данных</string>
   </property>
  </action>
  <action name="qSaveAllDB">
   <property name="text">
    <string>Сохранить все базы данных</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
